### DISCUSSION TITLE
Question 1

### DISCUSSION THREAD

**Discussion by Sreekar2610** [Sreekar2610/python#5] [Q&A] [answered]
In which year python was developed and by whom ??

**Comment by SandhyaMadhunagula** [comment, marked as answer]
Python was developed by Gudio Van Rossum in the year of 1991.

**Reply by Sreekar2610** [reply]
genius !!! correct answer

**Comment by Srilekhanalabolu** [comment]
Python was developed by Guido Van Rossum in the year of 1991.

**Comment by vishalbandaru02** [comment]
Python was developed by Guido Van Rossum in the year of 1991.
